### ISSUE TITLE
Final Project Review --zw287

### ISSUE THREAD

**Issue by RobinWu218** [kew96/LoanMachineLearning#14]
This project aims to build a model that predicts interest rates on loans based on many features of a customer including loan amount and loan history. The dataset is from Kaggle.

Things I like:
1. The project has a good selection of models. It includes simple linear models as well as XGBoost. 
2. Feature importance was analyzed in detail and the visualization was also great. It gives a good indication of which features are important.
3. I enjoyed the section about future improvements, especially the part about computational speed. It is a vital part for the model to be trained and deployed in real world.

Things to improve:
1. The visualizations could have been better. For example, figure 1 is very confusing to me with labels such as A,B,C,etc. Clearer visualizations with appropriate labels would improve the report by a lot.
2. In the section about data cleaning, you claim that 112 features are not important. This is a bit confusing as readers would not know why this is true. Providing some specific examples illustrating why would be very helpful.
3. Ridge Regression and linear regression are very similar. I would say it would be better to just use ridge regression as there is only one more regularization term and explore other models such as neural network. 

Overall, it is a solid project.
